rosdep/python.yaml

```diff
@@ -2980,6 +2980,16 @@ python-pycodestyle:
   gentoo: [dev-python/pycodestyle]
   rhel: [python-pycodestyle]
   ubuntu: [pycodestyle]
+python-pycryptodome:
+  alpine: [py3-pycryptodome]
```

Comment:
Did you intentionally add the package named `py3-` here?

Reply:
I noticed the same pattern for an entry close by: https://github.com/ros/rosdistro/blob/42df98bbea04a4a1260fb3a99db16f6148db8ba3/rosdep/python.yaml#L3412-L3413

But if preferred I can just remove the Python 2 entry for Alpine?

Reply:
I missed this when reviewing #21095. I've asked a [clarifying comment](https://github.com/ros/rosdistro/pull/21095#issuecomment-521340255). I think we can leave as-is since we'll need to address all instances of this if there is a mistake.